packages/dotcom-ui-header/src/__test__/components/__snapshots__/MainHeader.spec.tsx.snap

```diff
@@ -100,17 +100,13 @@ exports[`dotcom-ui-header/src/components/MainHeader renders ASK FT button 1`] =
             Subscribe
           </a>
           <a
-            aria-label="My F T"
-            className="o-header__top-icon-link o-header__top-icon-link--myft "
-            data-tour-stage="myFt"
-            data-trackable="my-ft"
-            href="/myft"
-            id="o-header-top-link-myft"
+            className="o-header__top-link ft-header__top-link--myaccount"
```

Comment:
**Reviewer notes:** The new css (eg `ft-header__top-link--myaccount`) that was added in the previous AB test PR is going to be revisited in a separate follow up PR to keep the scope of the PRs smaller